content/en/docs/reference/access-authn-authz/rbac.md

```diff
@@ -5,7 +5,7 @@ reviewers:
 - liggitt
 title: Using RBAC Authorization
 content_template: templates/concept
-aliases: [/rbac/]
+aliases: ["../../../../rbac/"]
```

Comment:
@jimangel I think this is one too many steps back, but it's working in the preview so 🤷‍♂ 

https://deploy-preview-19077--kubernetes-io-master-staging.netlify.com/rbac/